components/Signup/Page.vue

```diff
@@ -24,7 +23,8 @@
           <div :class="['strength-indicator', passwordStrengthClass]" :style="{ width: passwordStrengthPercentage + '%' }"></div>
```

Comment:
```suggestion
          <div :class="['strength-indicator', passwordStrengthClass]" :style="passwordStrengthStyle"></div>
```
we can still move this to a computed property **"width: passwordStrengthPercentage + '%'"**.
**passwordStrengthStyle** sounds good to me, however you may choose something else that feels more intuitive.

Reply:
no need to have another computed. just modified passwordStrengthPercentage and done